featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -461,7 +461,32 @@ def test_abides_by_max_depth_param(es):
         features = dfs_obj.build_features()
         for f in features:
             # last feature is identity feature which doesn't count
```

Comment:
Is this comment still valid/needed since it looks like we are checking this for all the features?